src/ssr-carousel-arrows.vue

```diff
@@ -30,12 +30,13 @@ export default
 	props:
 		index: Number
 		pages: Number
+		loop: Boolean
```

Comment:
@BKWLD/devs I'm adding looping support to vue-ssr-carousel.  Like where you can infinitely scroll right or left.  In Flickity this option is called [`wrapAround`](https://flickity.metafizzy.co/options.html#wraparound).  In vue-slick-carousel this is prop is called [`infinite`](https://github.com/gs-shop/vue-slick-carousel/blob/master/docs/API.md#props).

Thoughts on the prop being named `loop` since there doesn't seem to be a common convention for this name?

Reply:
I like `loop` best, too  👍